shaker/libs/pygit2_utils.py

```diff
@@ -100,3 +112,23 @@ def pygit2_check_credentials():
     message_ok = ("shaker.libs.util:pygit2_check_credentials: No credential problems found. ")
     shaker.libs.logger.Logger().debug(message_ok)
     return True
+
+
+def pygit2_check_ssh_agent():
```

Comment:
This would be clearer as `pygit2_agent_has_keys`: that way it's obvious that it returns a boolean when you're using it, and it's clear that a `True` return indicates that the agent has keys.